flow/Tracing.h

```diff
@@ -97,9 +97,9 @@ struct Span {
 // values in this enum can change without notice.
 enum class TracerType {
 	DISABLED = 0,
-	LOG_FILE = 1,
-	NETWORK_LOSSY = 2,
-	END = 3
+	NETWORK_LOSSY = 1,
+	END = 2,  // Any tracers that come after END will not be tested in simulation
```

Comment:
What exactly was the problem again? Maybe we can fix it and still exercise this code in simulation

Reply:
It's just extremely noisy, causing lots of additional `TraceEvent`s when enabled (calls into https://github.com/apple/foundationdb/blob/498a9740fe87acd9598da3224a8ce174725251b5/flow/Tracing.actor.cpp#L49 for every `Span` that gets destructed). I think it can be useful to enable selectively, say when debugging issues, but if it's causing simulation tests to fail because of too much output it is causing more harm than good.